### DISCUSSION TITLE
we would like to include your app in our app store

### DISCUSSION THREAD

**Discussion by r3sp0ns3** [dgroup/lazylead#531] [Ideas]
Hey, I found your lazylead app on GitHub. A really cool tool that we would like to include in our app store. Okay - who are we anyway?
We are Portal and we aim to create a digital ecosystem where private users can rent their own virtual servers in the cloud - their "Portals" - to do things they already do "online" today, like storing data, using apps, communicating, etc.
How this differs from today’s Internet: Portal prevents business models based on collecting user data and surveillance. Data mining giants cannot use our ecosystem for their own purposes. In addition, our goal is for apps to run both operating system-agnostic and end-device-agnostic to make users ’and developers’ lives easier.
We are currently in the pre-launch stage - the prototype of the Portal ecosystem is ready. What's missing are the first applications in our app store. This is where you would come in.

To make it as easy as possible for you to integrate your apps into our App Store, we use Docker container virtualization so you can freely choose your preferred language or frameworks.

More information on our project and how you can participate and benefit you find at getportal.org  Also, enter this code on preview.getportal.org to create your own Portal and take it for a test drive: hn0s-88ya

If you want to be part of it, I would be happy about a short message!

Greetings, Marcus

**Comment by dgroup** [comment]
Sure, that’s possible.
Will check next week.
